app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -48,16 +48,17 @@ export const CollectionPage = (): React.ReactElement => {
             </LayerContainer>
           </Box>
           <Spacing variant={PaddingSize.Wide2} />
-          <Text variant='title'>{collection.name}</Text>
           <ContainingView>
+          <Stack direction={Direction.Vertical} isFullWidth={true} childAlignment={Alignment.Center} padding={PaddingSize.Wide2}>
```

Comment:
this padding should probably be just horizontal. i would set veritcal to Wide or even just Default (there is too much space between the icon and the title)